### PR TITLE
query year dropdown

### PR DESCRIPTION
**Pull request by jonnamorrison**
pull request for the query year dropdown

### CODE REVIEW

File: CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx [hunk 1]
```diff
@@ -17,10 +21,18 @@ export default function CreateReport({ onDone }) {
...
     setCDCFile(e.target.files[0])
   }
+  const handleInputChange = (e) => {
+    setInputValue(e.target.value)
+  }
 
   const handleSubmit = async (e) => {
     e.preventDefault()
 
+    if (!inputValue) {
```
Comment (Carson21) [inline CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx:31]: should only check if a year is selected if the report is automatic. So after line 37, `if(isAutomatic)`.

**Review by p-zach — COMMENTED** [review]
nice
whoops this didnt approve it

**Review by p-zach — APPROVED** [review]
nice

**Review by solomonmcharo28 — APPROVED** [review]
This looks great. Well done!

**Pull request merged**